config.py

```diff
@@ -1,7 +1,7 @@
-consumer_key = "Your Consumer Key here"
-consumer_secret = "Your Consumer Secret here"
-access_token = "Your Access Token here"
-access_token_secret = "Your Access Secret here"
-volunteertwitterhandle = "Your Twitter handle"
+consumer_key = "c9pXLroOFv2y9wr4oLZh6Wlov"
```

Comment:
Please don't commit the credentials here.

Reply:
Sorry, I accidentally might have committed that because of the bad merge.